src/components/common/Input/index.js

```diff
@@ -1,21 +1,25 @@
 import React from 'react';
-import { string } from 'prop-types';
+import { string, bool, oneOfType, array } from 'prop-types';
 import { View, TextInput, Text } from 'react-native';
 import styles from './styles';
 
-const Input = ({ label, error, ...props }) => (
-  <View>
-    {label && <Text>{label}</Text>}
-    <View>
-      <TextInput style={styles.input} {...props} />
-      {error && <Text>{error}</Text>}
-    </View>
-  </View>
-);
+const Input = ({ label, invalid, error, ...props }) => {
+  const additionalStyles = invalid ? styles.inputBorderError : styles.inputBorder;
```

Comment:
What do you think about putting the input border as part of the input style, and then just do invalid && inputBorderError inside of the style array